scalopus_examples/src/showcase_count_events.cpp

```diff
@@ -27,62 +27,48 @@
   OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
   OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
-#include "scalopus_tracing/tracing.h"
```

Comment:
Has this file actually been renamed? I wonder why.

Reply:
Lol... thanks Github...

So I removed `scalopus_tracing/test/test_native_tracing_macros.cpp` and `scalopus_tracing/test/test_lttng_tracing_macros.cpp`. I still have that test, but the file is called `scalopus_tracing/test/test_tracing_macros.cpp` and just compiled twice linking against different libraries.

I also introduced this `showcase_count_events.cpp`, which apparently looks enough like the trace macro test to be identified as a move? :o And then the headers got sorted. And I changed the `""` to `<>` include.